src/Calculator/Views/GraphingCalculator/GraphingNumPad.xaml

```diff
@@ -0,0 +1,1194 @@
+
+        <controls:OperatorPanelListView x:Uid="ScientificOperatorPanel"
+                                        Grid.ColumnSpan="5"
+                                        Background="{ThemeResource AppOperatorPanelBackground}"
+                                        AutomationProperties.HeadingLevel="Level1">
+
+            <controls:OperatorPanelButton x:Name="TrigButton"
+                                          x:Uid="trigButton"
+                                          Style="{StaticResource OperatorPanelButtonStyle}"
+                                          AutomationProperties.AutomationId="trigButton"
+                                          Glyph="&#xF892;">
+
+                <controls:OperatorPanelButton.FlyoutMenu>
+                    <Flyout x:Name="Trigflyout"
+                            Windows10version1803:Placement="Bottom"
+                            Windows10version1809:AreOpenCloseAnimationsEnabled="False"
+                            Windows10version1809:Placement="BottomEdgeAlignedLeft"
+                            FlyoutPresenterStyle="{ThemeResource OperatorPanelFlyoutStyle}">
+                        <Grid x:Name="TrigGrid"
+                              MinWidth="258"
+                              MinHeight="96"
+                              XYFocusKeyboardNavigation="Enabled">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="1*"/>
+                                <RowDefinition Height="1*"/>
+                            </Grid.RowDefinitions>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="1*"/>
+                                <ColumnDefinition Width="1*"/>
+                                <ColumnDefinition Width="1*"/>
+                                <ColumnDefinition Width="1*"/>
+                            </Grid.ColumnDefinitions>
+                            <!--
+                                Because of the way we handle keyboard shortcuts, the IsEnabled property must be bound to a button outside the flyout.
+                                This is because the content from within the Flyout do not inherit the IsEnabled property of the flyout parent,
+                                causing the shortcut keys to be used when the control would normally be disabled.
+                            -->
+                            <ToggleButton x:Name="TrigShiftButton"
+                                          x:Uid="trigShiftButton"
+                                          Style="{StaticResource CaptionToggleEmphasizedButtonStyle}"
+                                          FontFamily="{StaticResource CalculatorFontFamily}"
+                                          AutomationProperties.AutomationId="trigShiftButton"
+                                          Checked="TrigFlyoutShift_Toggle"
+                                          Content="&#xF897;"
```

Comment:
>F897 [](start = 54, length = 4)

Looking at this, I don't think this glyph is indicative of inverse trig functions.

Reply:
Ah.  I see.  We're doing the same thing physical calculators do.